src/components/views/elements/EventTilePreview.tsx

```diff
@@ -118,13 +119,15 @@ export default class EventTilePreview extends React.Component<IProps, IState> {
     }
 
     public render() {
-        const event = this.fakeEvent(this.state);
-
         const className = classnames(this.props.className, {
             "mx_IRCLayout": this.props.layout == Layout.IRC,
             "mx_GroupLayout": this.props.layout == Layout.Group,
         });
 
+        if (!this.props.userId) return <div className={className} style={{ padding: '9px 0' }}><Spinner /></div>;
```

Comment:
Could the style here be moved to the `SCSS` file, please?

Reply:
I added a new `scss` file for the component. Can you check if it's referenced correctly? https://github.com/matrix-org/matrix-react-sdk/pull/6702/commits/d76adde1aba4fc6709dec5bb37a8db135a6d482e